### DISCUSSION TITLE
小狼毫RIME输入法双拼如何设置飞键？

### DISCUSSION THREAD

**Discussion by Chris-Zhang-China** [rime/home#1362] [Q&A]
某些双拼方案，如大牛双拼、飞猫双拼，含有大量的飞键键位。
请问这种双拼键位应该如何设置呢？
RIME的官方文档好像完全没有提到飞键的解决方案。
下面是飞猫双拼的键位图：

**Comment by j1g5awi** [comment]
只能自己生成码表，不能在 rime 里靠映射实现飞键
